### PR TITLE
CyberCat.ADBAppControl version 1.7.1

### PR DESCRIPTION
**Pull request by heymaad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36281)

### CODE REVIEW

File: manifests/c/Cyber/Cat/ADBAppControl/1.7.1/Cyber.Cat.ADBAppControl.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Cyber.Cat.ADBAppControl
```
Comment (ItzLevvie) [inline manifests/c/Cyber/Cat/ADBAppControl/1.7.1/Cyber.Cat.ADBAppControl.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: CyberCat.ADBAppControl
```

File: manifests/c/Cyber/Cat/ADBAppControl/1.7.1/Cyber.Cat.ADBAppControl.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Cyber.Cat.ADBAppControl
```
Comment (ItzLevvie) [inline manifests/c/Cyber/Cat/ADBAppControl/1.7.1/Cyber.Cat.ADBAppControl.installer.yaml:4]:
```suggestion
PackageIdentifier: CyberCat.ADBAppControl
```

File: manifests/c/Cyber/Cat/ADBAppControl/1.7.1/Cyber.Cat.ADBAppControl.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Cyber.Cat.ADBAppControl
```
Comment (ItzLevvie) [inline manifests/c/Cyber/Cat/ADBAppControl/1.7.1/Cyber.Cat.ADBAppControl.yaml:4]:
```suggestion
PackageIdentifier: CyberCat.ADBAppControl
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by heymaad — COMMENTED** [review]
Naming scheme corrections.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**